book/src/granular_lint_groups.md

```diff
@@ -0,0 +1,219 @@
+# More granular lint groups
+
+Clippy groups its lints into [9 primary categories](lints.md),
```

Comment:
I just noticed that `lints.md` doesn't have the nursery category documented, so that page actually only describes 8